lib/test.js

```diff
@@ -20,7 +20,7 @@ var commands = async (function(commandarr,catalog,exit_if_fail) {
 		}
 		try {
          //Usage of execSync is avoided as it not supported in windows. Hence, made the exec command asynchronous.
-			var out = await (childProcess.exec(command,{'stdio':['pipe','pipe','pipe']}));
+		  var out = spawn(command,{shell: true});;
```

Comment:
Used cross-spawn with shell option to be true. This is required for the windows machines.